Client/Frontend/Browser/Tabs/GroupedTabCell.swift

```diff
@@ -280,6 +279,7 @@ class GroupedTabContainerCell: UITableViewCell,
     func applyTheme(theme: Theme) {
         selectedView.backgroundColor = theme.colors.layer3
         collectionView.backgroundColor = theme.colors.layer5
+        collectionView.reloadData()
```

Comment:
We normally have `updateThemeApplicableSubviews` update any cells for us after the theme has changed to avoid reloading the collection views or table views entirely when the theme changes. If that doesn't work, then we should solve that problem instead of reloading the collection view.

Reply:
ok thank you i'm going to use that method instead.